preprocessing-service/preprocess.py

```diff
@@ -1,32 +1,72 @@
 # Standard Library
 import asyncio
+import json
 import logging
+import os
+import sys
 import time
 
 # Third Party
+from elasticsearch import AsyncElasticsearch
 from opni_proto.log_anomaly_payload_pb import Payload, PayloadList
 import pandas as pd
 from masker import LogMasker
 from opni_nats import NatsWrapper
 
 logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
-
+workload_parameters_dict = dict()
```

Comment:
this is fine for now, but maybe add a `TODO` bc in the future we should use the jetstream KV store instead.